app/subscriber/src/features/settings/MyMinisterSettings.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useLookup } from 'store/hooks';
+import { IOptionItem, RadioGroup, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+export const MyMinisterSettings: React.FC = () => {
+  const [{ ministers }] = useLookup();
+  const [myMinister, setMyMinister] = React.useState<string>();
+
+  React.useEffect(() => {
+    if (!!myMinister) localStorage.setItem('myMinister', myMinister);
+  }, [myMinister]);
+
+  const options = ministers.map((m) => {
+    return {
```

Comment:
It's probably better to create an `OptionItem` rather than force casting it to a `IOptionItem`